contracts/RelayerOracleConsumer.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "./interface/CLV2V3Interface.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract RelayerOracleConsumer is Ownable {
```

Comment:
I think we should include a short description about the contract as code documentation.